src/lib/RequestParser.js

```diff
@@ -0,0 +1,202 @@
+/* global Nimiq */
+/* global KeyStore */
+/* global Errors */
+/* global Utf8Tools */
+
+class RequestParser { // eslint-disable-line no-unused-vars
+    /**
+     * @param {any} appName
```

Comment:
one could improve type safety here by using unknown instead of any

Reply:
tsc is not happy with unknown. I'll leave it as any for now. All tests except parseTransaction should work with unknown, too